src/blister.c

```diff
@@ -284,17 +284,30 @@ Obj DoCopyBlist(Obj list, Int mut) {
   
 }
 
+Obj CopyBlistImm (
+    Obj                 list,
+    Int                 mut )
+{
+  return list;
+}
+		  
 Obj CopyBlist (
     Obj                 list,
     Int                 mut )
 {
-
-    /* don't change immutable objects                                      */
-    if ( ! IS_MUTABLE_OBJ(list) ) {
-        return list;
-    }
```

Comment:
Because immutable Blists now have their own function entered appropriately in the dispatch table. Perhaps `assert(IS_MUTABLE_OBJ(list));` should be added here.